src/components/TransactionBlock/HoneySwapLPTransactionBlock.tsx

```diff
@@ -185,7 +185,8 @@ const HoneySwapLPTransactionBlock = ({ id: transactionBlockId, errorMessages, va
       fromAssetIconUrl: selectedFromAsset?.logoURI,
       amount,
       accountType: selectedAccountType,
-      receiverAddress: receiverAddress ?? undefined,
+      receiverAddress:
+        (selectedReceiveAccountType === AccountTypes.Key ? providerAddress : accountAddress) ?? undefined,
```

Comment:
What's the UI result of this being undefined here?

Reply:
Nothing it won't let it pass through then during review